airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/common.py

```diff
@@ -65,3 +65,19 @@ def __str__(self) -> str:
 
     def __repr__(self) -> str:
         return str(self)
+
+
+def find_key_inside_schema(schema_item: Union[dict, list, str], key: str = "$ref") -> dict:
```

Comment:
What if it exists and has None value? I would suggest return boolean value and rename function to is_key_inside_schema. In case if it exists it would be True and False otherwise.

Reply:
I deliberately display an object (or a nested object) so that the developer can see it more clearly when displaying an error.
Changing the value in favor of the boolean value will lose the ability to see extended information about the test fail in the traceback.